packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -0,0 +1,1173 @@
+import '@aws-cdk/assert-internal/jest';
+import * as iam from '@aws-cdk/aws-iam';
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { EmrContainersStartJobRun, VirtualClusterInput, ReleaseLabel, ApplicationConfiguration, Classification } from '../../lib/emrcontainers/start-job-run';
+
+let stack: Stack;
+let clusterId: string;
+let defaultTask: EmrContainersStartJobRun;
+
+
+beforeEach(() => {
+  stack = new Stack();
+  clusterId = 'clusterId';
+  defaultTask = new EmrContainersStartJobRun(stack, 'Default EMR Containers Start Job Run', {
+    virtualCluster: VirtualClusterInput.fromTaskInput(sfn.TaskInput.fromText(clusterId)),
+    releaseLabel: ReleaseLabel.EMR_6_2_0,
+    jobDriver: {
+      sparkSubmitJobDriver: {
+        entryPoint: sfn.TaskInput.fromText('local:///usr/lib/spark/examples/src/main/python/pi.py'),
+        sparkSubmitParameters: '--conf spark.executor.instances=2',
+      },
+    },
+  });
```

Comment:
I meant that the props should be the constant. something like:
```ts
  defaultProps = {
    virtualCluster: VirtualClusterInput.fromTaskInput(sfn.TaskInput.fromText(clusterId)),
    releaseLabel: ReleaseLabel.EMR_6_2_0,
    jobDriver: {
      sparkSubmitJobDriver: {
        entryPoint: sfn.TaskInput.fromText('local:///usr/lib/spark/examples/src/main/python/pi.py'),
        sparkSubmitParameters: '--conf spark.executor.instances=2',
      },
    },
  };
```
then, you can either just directly use the props:
```ts
new EmrContainersStartJobRun(stack, 'Default EMR Containers Start Job Run', defaultProps);
```
or you can override/add specific properties

```ts
new EmrContainersStartJobRun(stack, 'Default EMR Containers Start Job Run', {
  ...defaultProps,
  optionalProperty: 'somevalue',
});
```